src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+        icon: /images/storage-icon.svg
+      - title: "Web3 Social Networks"
+        description: "TACo enables many-to-many data sharing, without a centralized authority that may harvest and monetize interpersonal communication."
+        icon: /images/social-icon.svg
+      - title: "Media content-gating"
+        description: "Whether it's a stream, a movie or a collectible, TACo enforces the decryption rights. 'Owning' an NFT now has meaning beyond a symbolic receipt."
+        icon: /images/media-icon.svg
+  - title: "The Web needs TACo"
+    cards:
+      - title: "Hyper-sensitive data"
+        description: "From reproductive healthcare to password recovery, TACo is used to ensure that high-stakes data sharing doesn’t involve trusting an intermediary."
+        icon: /images/locker-icon.svg
+      - title: "IoT & Connected Vehicles"
+        description: "TACo is the antidote to the interconnected hardware panopticon, in which every waking (and sleeping) moment of our lives is surveilled/harvested by data custodians."
+        icon: /images/iot-icon.svg
+      - title: "Activism & Human Rights"
+        description: "TACo protects the most vulnerable in society – those who may suffer  intimidation, censorship and violence as a consequence of their data being leaked."
+        icon: /images/activism-icon.svg
+tacoDiagramsInfo:
+  title: "Integrate a Privacy Layer with Nothing to Hide"
+  description: "TACo isn't just auditable, it's comprehensible. From high-level schematic to client code."
+  image: /images/taco-grid.svg
+  background: /images/glowing-background.png
+tacoDiagrams:
+  - title: "Generating a DKG Cohort"
+    preTitle: "learn"
+    description: "Independent nodes are sampled from the Threshold Network, forming a cohort that collectively generates a DKG public key."
+    image: /images/first-taco-diagram.svg
+    buttons:
+      - label: "nucypher/ferveo"
+        url: https://github.com/nucypher/ferveo
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher-contracts"
+        url: https://github.com/nucypher/nucypher-contracts
+        variant: EXTERNAL_OUTLINE
+  - title: "Encrypting Private Data"
+    preTitle: "learn"
+    description: "Data owners specify decryption conditions, embedded alongside the private data. The payload is encrypted with the DKG public key."
+    image: "/images/second-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+  - title: "Accessing Decrypted Data"
+    preTitle: "learn"
+    description: "Data requesters present proof to the cohort, who verify condition fulfillment before provisioning them with decryption material."
```

Comment:
```suggestion
    description: "Data requesters present proof to the cohort, who verify condition fulfilment before performing any decryption operation."
```

Reply:
Mm that slightly sounds like nodes have the power to decrypt unilaterally

Reply:
Perhaps. My issue was that "decryption material" could be interpreted as private key material. It's a tough one to word.

```suggestion
    description: "Data requesters present proof to the cohort, and each member verifies condition fulfilment before provisioning the requester with a decryption share; decryption requires a threshold of these shares."
```